programs/ssl/ssl_server2.c

```diff
@@ -729,6 +741,7 @@ sni_entry *sni_parse( char *sni_string )
             if( mbedtls_x509_crl_parse_file( new->crl, crl_file ) != 0 )
                 goto error;
         }
+#endif
```

Comment:
Minor: Please add the matching `#endif` comment `/* MBEDTLS_X509_CRL_PARSE_C */`. There are also a few other places where this happens.